src/_live-monitor.ts

```
@@ -91,61 +92,72 @@ export class LiveMonitor implements Disposable {
 			const sortedFiles = await sortFilesByTimestamp(filesToRead);
 
 			for (const file of sortedFiles) {
-				const content = await readFile(file, 'utf-8')
-					.catch(() => {
-						// Skip files that can't be read
-						return '';
-					});
+				const fileReader = Result.try({
+					try: async () => readFile(file, 'utf-8'),
+					catch: () => new Error('File read failed'),
+				});
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)